axe.d.ts

```diff
@@ -162,7 +167,7 @@ declare namespace axe {
     unsupported?: boolean;
   }
   interface AriaRoles {
-    type: AriaRolesType | DpubRolesType;
```

Comment:
I don't think this is correct. The `type` should be the `type` property from the role, which is just `structure` for each one. So you shouldn't need a new type for this as `AriaRoleTypes` covers strcuture.